ups/GalSim.table

```diff
@@ -0,0 +1,9 @@
+setupRequired(numpy)
+setupRequired(pyfits)
+setupRequired(boost)
+setupRequired(fftw)
+setupRequired(tmv)
+
+envPrepend(PYTHONPATH, ${PRODUCT_DIR})
+envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
```

Comment:
Whenever we set LD_LIBRARY_PATH, we should also set DYLD_LIBRARY_PATH unless the package is not intended to run on MacOS.